examples/waypoint_tracking_agent.py

```diff
@@ -0,0 +1,140 @@
+import logging
+import random
+
+import gym
+
+from smarts.core.agent import Agent, AgentSpec
+from smarts.core.agent_interface import AgentInterface, AgentType, DoneCriteria
+from smarts.core.sensors import Observation
+from smarts.core.utils.episodes import episodes
+from smarts.env.wrappers.single_agent import SingleAgent
+
+# The following ugliness was made necessary because the `aiohttp` #
+# dependency has an "examples" module too.  (See PR #1120.)
+if __name__ == "__main__":
+    from argument_parser import default_argument_parser
+else:
+    from .argument_parser import default_argument_parser
```

Comment:
Now that PR #1248 has been merged, I think you can simplify this back to just `from examples.argument_parser import default_argument_parser`.